services/ux-backend/main.go

```diff
@@ -61,8 +62,19 @@ func main() {
 		klog.Info("shutting down!")
 		os.Exit(-1)
 	}
+
+	// TODO: remove '/onboarding-tokens' in the future
 	http.HandleFunc("/onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
-		onboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
+		// Set the Deprecation header
+		w.Header().Set("Deprecation", "true") // Standard "Deprecation" header
+		w.Header().Set("Link", "/onboarding-tokens/client-token; rel=\"alternate\"")
+		clienttoken.HandleMessage(w, r, config.tokenLifetimeInHours)
+	})
+	http.HandleFunc("/onboarding-tokens/client-token", func(w http.ResponseWriter, r *http.Request) {
+		clienttoken.HandleMessage(w, r, config.tokenLifetimeInHours)
+	})
+	http.HandleFunc("/onboarding-tokens/peer-token", func(w http.ResponseWriter, r *http.Request) {
+		peertoken.HandleMessage(w, r, config.tokenLifetimeInHours)
 	})
```

Comment:
As of the new path design, onboardingtokens becomes more of a subject then a resource. Maybe we should rename the new paths to `/onboarding/client-tokens` and `/onboarding/peer-tokens`. It will also help in removing the confusion around the old deprecated path.